test/integration/app/prompted_nobuild.js

```diff
@@ -26,6 +26,7 @@ var rimraf = require('rimraf');
 var fs = require('fs');
 
 var appGeneratorPath = path.join(__dirname, '../../../app');
+var extendedTimeout = 300000
```

Comment:
Right, so when you select the `Endpoints from swagger file` option with a Swagger file, a companion iOS SDK is created. Ricardo and I thought that was good behavior since the iOS SDK can communicate directly with the generated endpoints. A side effect is the tests take a little bit longer because we are performing a network call. Not sure if we should change generator, tests, or just let it be?